### ISSUE TITLE
Step 4: "Use of uninitialized value in hash element"

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#145]
**Reported by bli on 31 Oct 2005 08:32 UTC**
```
Got the following warning message after Image Creation
Step:

Use of uninitialized value in hash element at
/usr/lib/perl5/vendor_perl/5.8.0/i386-linux-thread-multi/Tk/Trace.pm
line 181.

Trunk r3897, RHEL4u2 x86.
```

**Comment by dikim33** [comment]
**Comment by bli on 2 Dec 2005 08:47 UTC**
```

This seems to be a problem with perl-Tk, might be fixed with
a newer version: perl-Tk-804.027.
```

**Comment by dikim33** [comment]
**Comment by bli on 3 Jan 2006 08:08 UTC**
```

A possible solution is mentioned here:

http://www.archivum.info/comp.lang.perl.tk/2005-06/msg00000.html
```

**Comment by dikim33** [comment]
**Comment by bli on 19 Feb 2006 05:47 UTC**
```

Fix integrated into systeminstaller-oscar-2.1-1.noarch.rpm
which was checked into OSCAR trunk r4373.
```
